searchTest.js

```diff
@@ -1,41 +1,35 @@
-import { Selector } from 'testcafe';
+import { Selector } from 'testcafe'
 
 fixture `Google Test Search`
-    .page `https://www.google.com`;
+    .page `https://www.google.com`
+
+test('it returns theartofeducation.edu when searching for the art of education', async t => {
+    let googleSearchBox = ".gLFyf"
+    let googleSearchButton = ".gNO89b"
 
-test('Search for AOEU', async t => {
     await t
-        //Type "The Art of Education University in the text field"
-        .typeText('.gLFyf', 'The Art of Education University')
-        //Clicks the google search button
-        .click('.gNO89b')
-        //Clicks "The Art of Education University" link
+        .typeText(googleSearchBox, 'The Art of Education University')
+        .click(googleSearchButton)
         .click('.yuRUbf')
```

Comment:
could also give `.yuRUbf` a more meaningful name like` googleSearchResult`?

Reply:
Yep! Thank you.